### PR TITLE
Add IBC E2E test with Hermes

### LINKED ISSUE #773: End-to-end IBC tests using the Hermes relayer
Before finalizing the Namada IBC integration, we need to have e2e tests with the Hermes relayer, as this is what validators & relayer operators will be using.

### PR DESCRIPTION
**Pull request by yito88**
## Describe your changes
closes #773
depends on #2237
Add a new IBC E2E test with Hermes relaying

## Indicate on which release or other PRs this topic is based on
~~#1784~~ ~~`0.28.0`~~ `0.29.0`

## Checklist before merging to `draft`
- [x] I have added a changelog
- [x] Git history is in acceptable state

### CODE REVIEW

File: tests/src/e2e/setup.rs [hunk 1]
```diff
@@ -1138,6 +1138,93 @@ pub fn sleep(seconds: u64) {
...
+    }
+
+    Ok(())
+}
+
+pub fn run_hermes_cmd<I, S>(
+    test: &Test,
+    args: I,
+    timeout_sec: Option<u64>,
+) -> Result<NamadaCmd>
```
Comment (sug0) [inline tests/src/e2e/setup.rs:1169]: hm this is kinda hacky. oh well :D

**Review by sug0 — APPROVED** [review]

**Comment by tzemanovic** [comment]
the e2e test failed on 1st attempt, retrying

**Review by sug0 — CHANGES_REQUESTED** [review]
ah you're right, oops. looks like it failed again.

**Comment by yito88** [comment]
ah, it should fail because Hermes for Namada 0.28.0 ([1.7.1-namada-beta5](https://github.com/heliaxdev/hermes/releases/tag/v1.7.1-namada-beta5)) is required.
For CI, #2237 is required to build Hermes with the target Namada.

**Comment by sug0** [comment]
I see

**Review by sug0 — APPROVED** [review]

**Comment by yito88** [comment]
Rebased. The E2E test passed with Hermes `1.7.4-namada-beta6`.

**Pull request merged**